flax/nn/base.py

```diff
@@ -1010,10 +1010,9 @@ def store(self, value):
     # is ill-defined (eg. what does vmap of BatchNorm do?).
     # TODO(jheek): Add doc guide on combining jax transforms and state.
     # TODO(jheek): Should some transformations be excempt from this error?
-    master = utils._tracer_of_value(value)
-    value_level = master.level if master else float('-inf')
-    state_level = self._master.level if self._master else float('-inf')
-    if value_level > state_level:
+    value_level = utils._level_of_value(value)
+    print('state_level, value_level', self._master_level, value_level, value)
```

Comment:
Do we really want to always print these values?

Reply:
Yo - just checking in again about this PR ;)

Reply:
Good catch I left some debug code in my PR